recipes/pykdtree/build.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+if [[ $(uname) == Darwin ]]; then
+    USE_OMP=0
```

Comment:
So, we could use the Continuum `gcc` package here for Mac so that you could use OpenMP. That seems like a good idea if that is what you are asking. Alternatively, you could try to use Mac's ancient `gcc` not great, but if it works it will save you a dependency.